Netcracker/KubeMarine
# [CPDEV-94448] Improve check_paas checks and logging

### Description
* Some PaaS checks fail without detailed reason:
    * 207 Pods Condition
    * 220 Control Plane Configuration Status
    * 224 Calico Configuration Check
* Missed some default plugins in some checks:
    * 222 Default Services Configuration Status
    * 223 Default Services Health Status
* Calico env configuration is not fully checked.

### Solution
* Added DEBUG logs when comparing configurations to show the difference.
* Added missed plugins and services to the checks.
* Calico env variables for calico-node and calico-typha are compared for absent/extra/changed variables.

### Test Cases

**TestCase 1**

Check control plane configuration.

Steps:

1. Install any cluster.
2. Change extraArgs/extraVolumes for services.kubeadm.apiServer or other control plane configuration, or change services.kubeadm.kubernetesVersion in the `cluster.yaml`.
3. Run `check_paas`.

Results:

| Before | After |
| ------ | ------ |
| 220 check fails without clear reason | 220 fails with reason in DEBUG logs |

**TestCase 2**

Check plugins health / configuration.

Steps:

1. Install any cluster with necessary enabled plugins or their services.
2. Change version of `calico-typha`, `local-path-provisioner`,
    `kubernetes-dashboard` in the `cluster.yaml`, or change `dashboard-metrics-scraper` image version in the `cluster.yaml`, or break the correspond services on the cluster in some way.
3. Run `check_paas`.

Results:

| Before | After |
| ------ | ------ |
| 222 and 223 checks succeed | 222 or 223 checks fail with reason in HINT |

**TestCase 3**

Check calico env configuration.

Steps:

1. Install any cluster.
2. Add some `plugins.calico.env` in the `cluster.yaml`.
3. Run `check_paas`.

Results:

| Before | After |
| ------ | ------ |
| 224 check succeeds | 224 check fails with reason in DEBUG logs |

### Checklist
